test/services/teams/helper/helper.user.js

```diff
@@ -57,31 +65,34 @@ const createUser = async (userId, roleName = 'student', schoolId = '0000d186816a
 		schoolId,
 		firstName: userId,
 		lastName: 'GeneratedTestUser',
-		roles: [
-			role._id,
-		],
+		roles: [role._id],
 	});
 };
 
-const createAccount = (userId) => accountModel.create({
-	username: userId + AT,
-	password: TEST_HASH,
-	userId,
-	activated: true,
-});
+const createAccount = (userId) =>
+	accountModel.create({
+		username: userId + AT,
+		password: TEST_HASH,
+		userId,
+		activated: true,
+	});
 
 const setupUser = async (roleName, schoolId) => {
 	const userId = new ObjectId();
 	const user = await createUser(userId, roleName, schoolId);
 	const account = await createAccount(user._id);
 	const accessToken = await getToken(account);
 	return {
-		userId: user._id, account, user, accessToken,
+		userId: user._id,
+		account,
+		user,
+		accessToken,
 	};
 };
 
 const deleteUser = async (userId) => {
-	if (typeof userId === 'object' && userId.userId !== undefined) userId = userId.userId;
+	if (typeof userId === 'object' && userId.userId !== undefined)
+		userId = userId.userId;
```

Comment:
Codacy found an issue: [Assignment to function parameter 'userId'.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=5987920)